### ISSUE TITLE
ret paladin not recomending free divine storms instead showing to wait on cd on other abilities

### ISSUE THREAD

**Issue by ZethenZynx** [Hekili/hekili#3398]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

At some points in the rotation it will show a cooldown on an ability while a free divine storm (or more) is available. Most of the time the cooldown is templar strikes but can be other abilities like blade of justice. This did not happen before the pre-patch, there were no "pauses" in rotation. Sorry about the wrong raidbot sim, I did not log out before sim, this one should be correct. To answer your next question about mistaking Echoes of Wrath with a free Divine Storm, I took off teir to not have that set bonus and the problem still persists when a free divine storm happens from empyrean power.

### How to Reproduce

take the empyrean power talent, to reproduce dps aoe target dummies and when a free divine storm happens it will show a cooldown on an ability instead. now maybe this does not happen the first time a free divine storm happens, i am not sure but it does happen after following the recommended rotation

### Snapshot (Link)

https://pastebin.com/NCZFHvnF

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/7CedAGbunYtVJrFCFE1Zhj

### Additional Information

i really tried to answer your questions correctly, this is the same talent build as before and i made sure it was the new version of hekili as of today 7/26/24. If you are not able to reproduce this issue that I am getting, I could share my screen with you in discord if that would help

CYEALSUfbn0RkIzCqAITj7R8pAAAAwAAMZZZbmZbZxMWmZmlF2GAAAAAAGqZmZ2MwMYMLjhZGYWmZZbMwMMGLbYAAAAAAAAA

### Contact Information

zethenzynx on discord

**Comment by Hekili** [comment]
You could've added to the same ticket...

**tl;dr -- You configured Templar's Verdict to be unusable with multiple targets, but took the Divine Arbiter talent.**

![image](https://github.com/user-attachments/assets/960da3c7-2c08-4443-afb1-dd3e408c9c2d)

Set Maximum Targets to 0.

---

```
player_buffs:
    32223 - crusader_aura                            -   1 - 3600.00
   245686 - *fashionable                             -   1 - 3600.00
   430666 - *sign_of_awakened_storms                 -   1 - 3600.00
   430668 - *sign_of_awakened_embers                 -   1 - 3600.00
   430669 - *sign_of_awakened_dreams                 -   1 - 3600.00
   186403 - *sign_of_battle                          -   1 - 3600.00
   170869 - *blazing_wings                           -   1 - 104.30
   440839 - *kindness_of_chiji                       -   1 - 28009.36
   426339 - *igiras_cruel_nightmare                  -   1 - 3600.00
   415603 - *encapsulated_destiny                    -   1 - 3600.00
   406975 - divine_arbiter                           -  25 - 28.70 
   385126 - blessing_of_dusk                         -   1 - 0.78  
   326733 - empyrean_power                           -   1 - 10.34 
   385127 - blessing_of_dawn                         -   2 - 18.58 
```

You do have Empyrean Power active in your snapshot this time.

```
        26.  variable ( finishers - 1 )
         - variable.ds_castable[false] will check this script entry ( Retribution:finishers:1 )
```

This variable (Divine Storm castable) is returning false, but it has to be true for Divine Storm to be used.  That variable evaluates to:

```
( spell_targets.divine_storm[5] >= 3 | spell_targets.divine_storm[5] >= 2 & ! talent.divine_arbiter[false] | buff.empyrean_power.up[true] ) & ! buff.empyrean_legacy.up[false]& ! ( buff.divine_arbiter.up[true] & buff.divine_arbiter.stack[25] > 24 )
```

The first grouping succeeds for two reasons:  You have 3+ targets and Empyrean Power is up.
The next clause requires Empyrean Legacy to be inactive, so it succeeds.
The last clause fails because it requires that Divine Arbiter is inactive or at fewer than 25 stacks.

Divine Arbiter:  At 25 stacks your next damaging **single target** Holy Power ability causes (375% of Attack power) Holystrike damage to your primary target and (75% of Attack power) Holystrike damage to enemies within 6 yds.

It's avoiding Divine Storm because you should be casting a buffed single-target Holy Power ability, i.e. Templar's Verdict.  So why isn't it recommending Templar's Verdict?

```
        30.  templars_verdict ( finishers - 5 ) - ability disabled ( active_enemies[5] is more than ability's maximum targets [1] )
```

You have set Templar's Verdict to be disallowed when you have more than 1 target.  You should turn that off.
